mesonbuild/compilers/mixins/gnu.py

```diff
@@ -306,7 +306,7 @@ def get_include_args(self, path: str, is_system: bool) -> T.List[str]:
 
     @classmethod
     def use_linker_args(cls, linker: str) -> T.List[str]:
-        if linker not in {'gold', 'bfd', 'lld'}:
+        if linker not in {'gold', 'bfd', 'lld', 'qcld'}:
```

Comment:
In what way is qcld different than the other linkers?

Reply:
Basically just asking so you can add a comment about it here.

Reply:
Thanks. I added a comment here.